src/graphql/schema.ts

```diff
@@ -0,0 +1,30 @@
+export const typeDefs = `#graphql
+  type Query {
+    hello: String
+  }
+
+  input UserInput {
+    name: String!
+    email: String!
+    password: String!
+    birthDate: String!
+  }
+
+  type User {
+    id: ID!
+    name: String!
+    email: String!
+    birthDate: String!
+  }
+
+  type Mutation {
+    createUser(data: UserInput!): User
```

Comment:
```suggestion
    createUser(data: UserInput!): User!
```

Acho que aqui a ideia é sempre retornar um usuário, então acredito que seja bom colocar o !